deps/Makefile

```diff
@@ -171,9 +171,11 @@ ifeq ($(USE_SYSTEM_LIBSSH2), 0)
 STAGE2_DEPS += libssh2
 endif
 
+ifneq ($(OS), WINNT)
```

Comment:
since the makefile dependencies are not conditional on OS, this ifeq won't prevent curl from building on windows